src/ClusterBootstrap/cloud_init_aztools.py

```diff
@@ -546,17 +594,112 @@ def get_deployed_cluster_info(config, args):
     for name, spec in vminfo.items():
         brief_spec = {}
         brief_spec["admin_username"] = spec["osProfile"]["adminUsername"]
+        brief_spec["vm_size"] = spec["hardwareProfile"]["vmSize"]
         brief_spec["public_ip"] = spec["publicIps"]
         brief_spec["private_ip"] = spec["privateIps"]
         brief_spec["fqdns"] = spec["fqdns"]
         brief_spec["role"] = spec["tags"]["role"].split('-')
         brief[name] = brief_spec
-    args.output = "status.yaml" if not args.output else args.output
-    with open(args.output, "w") as wf:
+    output_file = "status.yaml" if not args.output else args.output
+    with open(output_file, "w") as wf:
         yaml.safe_dump({"machines": brief}, wf)
 
 
-def run_command(command, config, args, nargs):
+def get_k8s_node_list_under_condition(k8scmd):
+    output = exec_cmd_local(k8scmd)
+    nodes = output.split()
+    return nodes
+
+def delete_k8s_nodes(node_list, args):
+    for node in node_list:
+        cmd = './ctl.py kubectl delete node {}'.format(node)
```

Comment:
We are in serious need of moving `kubectl` function into a common utils file that can be imported and used readily. We can do this in a later PR, but we must do it.